### PR TITLE
Travis CI and Packaging Up

### PR DESCRIPTION
**Pull request by cleborys**
Ready for Travis ContinuousIntegration with status badge (currently shows the status of my fork's master branch until you set it up yourself). The actual plus here is that it's now an installable `pip` package and easy to deploy, see instructions in readme.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  chrome: stable
+  firefox: latest
+python:
+  - "3.6"
+# install dependencies
+before_install:
+  - wget https://github.com/mozilla/geckodriver/releases/download/v0.24.0/geckodriver-v0.24.0-linux64.tar.gz
+  - wget https://chromedriver.storage.googleapis.com/73.0.3683.68/chromedriver_linux64.zip
+  - mkdir geckodriver
+  - mkdir chromedriver
```
Comment (FjolleJagt) [inline .travis.yml]: These two can be on the same line

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  - tar -xzf geckodriver-v0.24.0-linux64.tar.gz -C geckodriver
+  - unzip chromedriver_linux64.zip -d chromedriver
+  - export PATH=$PATH:$PWD/geckodriver
+  - export PATH=$PATH:$PWD/chromedriver
```
Comment (FjolleJagt) [inline .travis.yml]: As can these two

File: requirements.txt [hunk 3]
```diff
@@ -4,8 +4,8 @@ chardet==3.0.4
 coverage==4.5.3
 easyimap==0.6.3
 EasyProcess==0.2.5
+kbhff==0.0.0
 more-itertools==7.0.0
-pkg-resources==0.0.0
 pluggy==0.9.0
```
Comment (FjolleJagt) [inline requirements.txt:8]: It seems there's [a CVE](https://nvd.nist.gov/vuln/detail/CVE-2019-11324) recommending to upgrade urllib to `urllib3>=1.24.2`.

**Review by FjolleJagt — APPROVED** [review]
Looks good!

**Comment by cleborys** [comment]
Also includes the pull requests approved on my fork.

**Pull request merged**